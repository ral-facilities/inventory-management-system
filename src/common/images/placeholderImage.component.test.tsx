import { render } from '@testing-library/react';
import PlaceholderImage from './placeholderImage.component'; // Adjust the import path as necessary

describe('PlaceholderImage Component', () => {
  it('matches the snapshot', () => {
    const { asFragment } = render(
      <PlaceholderImage maxWidth="300px" maxHeight="200px" />
    );
    // Take a snapshot of the rendered component
    expect(asFragment()).toMatchSnapshot();
  });
});
