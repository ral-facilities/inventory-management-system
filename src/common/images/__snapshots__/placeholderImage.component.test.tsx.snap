// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`PlaceholderImage Component > matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-q2z5p8"
  >
    <h5
      class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
    >
      No Image
    </h5>
  </div>
</DocumentFragment>
`;
